plugins/usbdmx/LibUsbAdaptor.cpp

```diff
@@ -180,7 +180,12 @@ int BaseLibUsbAdaptor::ClaimInterface(libusb_device_handle *dev,
 int BaseLibUsbAdaptor::DetachKernelDriver(libusb_device_handle *dev,
                                           int interface_number) {
   if (libusb_kernel_driver_active(dev, interface_number)) {
-    return libusb_detach_kernel_driver(dev, interface_number);
+    int r = libusb_detach_kernel_driver(dev, interface_number);
+    if (r) {
+      OLA_WARN << "libusb_detach_kernel_driver failed for: " << dev
+               << ": " << LibUsbAdaptor::ErrorCodeToString(r);;
```

Comment:
Changed all 4 to commas, we need something to separate the device from the error.